microsoft-edge/webview2/how-to/debug-visual-studio.md

```diff
@@ -35,59 +35,96 @@ To debug your code, first install and set up the script debugger tools in Visual
 
-1. In Visual Studio, under **Workloads**, select the **Desktop Development with C++** setting:
+1. In Visual Studio, under **Workloads**, select the **Desktop Development with C++** workload, so that a checkmark appears:
 
    ![Visual Studio Modifying Workloads Screen](./debug-visual-studio-images/workloads.png)
 
 1. Select **Individual components**, at the top.
 
 1. In the search box, enter `JavaScript diagnostics`.
 
-1. Select the **JavaScript diagnostics** setting.
-
-1. Click **Modify**.
+1. Select the **JavaScript diagnostics** checkbox:
 
    ![Visual Studio: Modifying values in the 'Individual components' tab](./debug-visual-studio-images/indiv-comp.png)
 
+1. Click the **Modify** button.
+
 
 <!-- ====================================================================== -->
 ## Enable script debugging for WebView2 apps
 
-Second, enable script debugging for WebView2 apps, as follows.
+Second, enable script debugging for WebView2 apps, as follows.  You can open your own WebView2 app in Visual Studio 2019 or 2022.  The following steps walk you through:
+* Opening the Win32APISample sample in Visual Studio 2022.
+* Working within your own working branch of the WebView2Samples repo, where some of the project files are modified during updating to Visual Studio 2022.
 
-1. Right-click your WebView2 project, and then select **Properties**.
+1. If not done already, do the steps in [Win32 sample app](../samples/webview2apissample.md) so that the WebView2APISample app builds and runs.  Or open your own WebView2 app project in Visual Studio.
 
-1. Under the **Configuration Properties**, select **Debugging**.
+1. In Visual Studio, in Solution Explorer, right-click the **WebView2APISample** project (or your own project), and then select **Properties**.
 
-1. Under the **Debugger Type**, select **JavaScript (WebView2)**.
+1. Under **Configuration Properties**, select **Debugging**.
 
-   ![The 'Debugging' configuration property in Visual Studio](./debug-visual-studio-images/enb-js.png)
+1. For **Debugger Type**, select **JavaScript (WebView2)**, and then click the **OK** button:
+
+   ![The 'Debugging' configuration property in Visual Studio](./debug-visual-studio-images/enb-js.png)<!-- WebView2APISample -->
 
 
 <!-- ====================================================================== -->
 ## Debug your WebView2 app
 
 After doing the above setup, debug your WebView2 app, as follows.
 
-1. To set a breakpoint in your source code, hover to the left of the line number, and click to set a breakpoint.  The JS/TS debug adapter doesn't perform source path mapping.  You must open the exact same path associated with your WebView2.
+1. To set a breakpoint in your source code, hover to the left of the line number, and click to set a breakpoint.  For example, open `WebView2Samples\SampleApps\WebView2APISample\assets\ScenarioJavaScriptDebugIndex.js`, and set a breakpoint on line 2: `console.log("onAddClick+");`, in the body of `function onAddClick() {`:
```

Comment:
not clear that setting this breakpoint does anything, in these steps
@victorhuangwq

Reply:
The breakpoint only triggers when this section of the code is executed. I believe that this should be triggered upon clicking the "Add a new item" button in step 3. 

But I don't think there's a step that asks the user to click on it, and thus this breakpoints functionality is demonstrated.